template-processors/applier/README.md

```diff
@@ -0,0 +1,13 @@
+# OpenShift Applier Template Processor
```

Comment:
Please also update the main readme file and add it to the list of provided templating engines.

Reply:
Done.